README.md

```diff
@@ -70,8 +70,12 @@ pip install rebuff
 ```python
 from rebuff import Rebuff
 
-# Set up Rebuff with your playground.rebuff.ai API key, or self-host Rebuff 
-rb = Rebuff(api_token="<your_rebuff_api_token>", api_url="https://alpha.rebuff.ai")
+# For a quick start, use our hosted rebuff server with your user's specific API token
+# Your `<your_rebuff_api_token>` can be found here: https://www.rebuff.ai/playground#add-to-app
+
+# Alternatively, you can self host your own rebuff server: https://github.com/protectai/rebuff#self-hosting
+
+rb = Rebuff(api_token="<your_rebuff_api_token>", api_url="https://www.rebuff.ai/")
```

Comment:
It looks like we need to remove the `/` from the end of `https://www.rebuff.ai`. The SDK appends this slash with the route. The other examples in this PR will need this update as well.